meson.build

```diff
@@ -457,6 +459,7 @@ kmod_sources = files(
 kmod = executable(
   'kmod',
   kmod_sources,
+  c_args : '-DHAVE_FINIT_MODULE=0',
```

Comment:
we need at least a better commit message. It doesn't really make sense to have HAVE_FINIT_MODULE=0 hardcoded. In that case why don't we **drop** it and always use the call via syscall()?